lib/schemes/nested.js

```
@@ -3,31 +3,22 @@ var path = require('path'),
     bemNaming = require('bem-naming');
 
 module.exports = {
-    path: function(entity, tech, options) {
+    path: function(cell, options) {
```

Maybe we should throw an exception if there isn't BemCell?